packages/calendar/react/src/components/action-pane/availability.tsx

```diff
@@ -172,78 +205,90 @@ const Availability: React.FC<{
 
       <div className="pt-2">
         <div className="h-48 overflow-auto rounded border border-gray-50 bg-white py-1 pl-3 pr-1">
-          <div className="flex p-1 text-gray-700">
-            <span className="flex-1">
-              {selectedSlots.length
-                ? getCopyTitle(preference)
-                : "Selected slots will appear here"}
-            </span>
-            <button
-              className="h-7 w-7 pl-1"
-              title="copy"
-              disabled={selectedSlots.length === 0}
-              onClick={() => {
-                setCopied(true);
-                setTimeout(() => setCopied(false), 2000);
-                copyToClipboard(selectedSlots, preference);
-                toast("Saved to clipboard!", {
-                  position: "bottom-center",
-                  hideProgressBar: true,
-                  autoClose: 1000,
-                  type: "info",
-                });
-              }}
-            >
-              {copied ? (
-                <BiCheckCircle className="h-5 w-5 text-green-500" />
-              ) : (
-                <BiCopy className="h-5 w-5" />
-              )}
-            </button>
-          </div>
-          {Object.keys(getGroupedSlots(selectedSlots)).map((key, idx) => {
-            const events: EventInput[] = getGroupedSlots(selectedSlots)[key];
-            const firstEvent = events[0];
-            if (!firstEvent) return null;
+          {!selectedSlots.length ? (
+            <div className="flex justify-between">
+              <div
+                className="color-gray-200 text-sm"
+                style={{ paddingTop: "2px" }}
+              >
+                Selected slots will appear here
+              </div>
+              {copyButton}
+            </div>
+          ) : (
+            <div>
+              <div className="flex justify-between pb-3">
+                {formatting?.introSentence ??
+                getCopyTitle(preference?.formatLanguage) ? (
+                  <div className="text-sm" style={{ paddingTop: "2px" }}>
+                    <span className="color-gray-700">
+                      {formatting?.introSentence ??
+                        getCopyTitle(preference?.formatLanguage)}
```

Comment:
Looks like it's the same as in lines [221-222](https://github.com/agreeto/agreeto/pull/15/files#diff-ae40774dc406ba05e0a9b78d9622fa6921af1219c18a951efc612a1bd795d584R221-R222) and [238-239](https://github.com/agreeto/agreeto/pull/15/files#diff-ae40774dc406ba05e0a9b78d9622fa6921af1219c18a951efc612a1bd795d584R238-R239)?

could extract to constants to improve readbility